frontend/src/employee-frontend/types/vasu.ts

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { JsonOf } from '../../lib-common/json'
+import { UUID } from './index'
+
+export enum VasuDocumentState {
+  DRAFT = 'DRAFT',
+  CREATED = 'CREATED',
+  REVIEWED = 'REVIEWED',
+  CLOSED = 'CLOSED'
+}
```

Comment:
hmm haven't seen this before, no idea how it compares to `type VasuDocumentState = 'DRAFT' | 'CREATED' | ...`

Reply:
Enum ends up in the generated js whereas `type` is just type info. Switched to type.